docs/managing-e2e-tests.md

```diff
@@ -23,106 +24,140 @@ It contains a list of jobs such as:
+  testgrid-num-columns-recent: "20"
+  testgrid-num-failures-to-alert: "4"
+  testgrid-alert-stale-results-hours: "8"
 ```
-can exists in both the `sig-cluster-lifecycle-all` and `sig-release-master-blocking` dashboards. Essentially both tests would show details about the same test job called `ci-kubernetes-e2e-kubeadm-kind-1-12`.
+
+These annotations configure the testgrid entries for this job from the job configuration itself.
+They contain information such as dashboards where this job will appear, tab name, where to send email alerts,
+description and other.
+
+For more information about configuring testgrid see [this page](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md).
 
 ### Updates to kubeadm tests
 
 Before each new Kubernetes release and during the second month of the [release-cycle](https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md), a set of manual actions have to be performed, so that the kubeadm e2e tests are up to date with the new release.
 
-The operation can be broken down in 3 steps:
-1) Updating the kubeadm test jobs and parameters.
-2) Updating testgrid to reflect the changes.
-3) Finalizing the changes.
+The operation can be broken down into:
+- Sending a PR for updating kinder workflows.
+- Sending a PR for updating test jobs in kubernetes/test-infra.
+
+Ideally both PRs should be merged around the same time, so that there are no failed runs.
 
-There is work in progress in test-infra on tools that can hopefully automate this process.
+The idea is to remove old tests and add new ones. For example. if Kubernetes 1.15 is about to be released soon,
+this means that the `15 - 3 = 12` MINOR version would go outside of the support skew of the project. `1.12` jobs
+have to be removed and `1.15` jobs have to be added. The project would support version `15 - 2 = 13` as the minimum.
 
-#### Updating test jobs
+#### Updating kinder workflows
+
+Kinder is deployment tool that currently resides in the kubernetes/kubeadm repository.
+It contains a folder with workflows that need update each cycle:
+https://github.com/kubernetes/kubeadm/blob/master/kinder/ci/workflows/
+
+The summary of the actions is the following:
+- If there are `1.12 -> 1.13` upgrade jobs leave them, because upgrades to `1.13` still have to be supported.
+- Remove `1.13 on 1.12` jobs.
+- Remove plain `1.12` jobs.
+- Add upgrade jobs for `1.14 -> 1.15` and `1.15 on 1.14` skew jobs.
+- Add plain `1.15` jobs.
+- Make sure that workflows include the correct `ci/latest-x` labels.
+- Make sure that workflows have the correct testgrid URLs.
 
-This document will cover information on how to perform updates on these 3 files:
-- [kubeadm-kind.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml "kubeadm-kind.yaml")
+#### Updating test jobs in kubernetes/test-infra
+
+This document will cover information on how to perform updates on these files:
```

Comment:
I think we should link to  https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md instead of maintaining another list of test

Reply:
https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md is outdated.
i think instead we should link from https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md to this document that is specific for tests.